### PR TITLE
Inline references for 0x04g

### PR DESCRIPTION
**Pull request by TheDauntless**
Inline references for 0x04g (#590 )

### CODE REVIEW

File: Document/0x04g-Testing-Cryptography.md [hunk 1]
```diff
@@ -66,10 +58,10 @@ The source code should be checked that cryptographic algorithms are up to date a
 
 Inspect the source code to identify the instances of cryptographic algorithms throughout the application, and look for known weak ones, such as:
```
Comment (Sjord) [inline Document/0x04g-Testing-Cryptography.md]: Maybe the ENISA report is better linked here under "known weak ones".

File: Document/0x04g-Testing-Cryptography.md [hunk 2]
```diff
@@ -89,7 +81,7 @@ Other uses of cryptography require careful adherence to best practices:
 
 #### Remediation
 
-Periodically ensure that the cryptography has not become obsolete. Some older algorithms, once thought to require years of computing time, can now be broken in days or hours. This includes MD4, MD5, SHA1, DES, and other algorithms that were once considered as strong. Examples of currently recommended algorithms<sup>[1], [2]</sup>:
+Periodically ensure that the cryptography has not become obsolete. Some older algorithms, once thought to require years of computing time, can now be broken in days or hours. This includes MD4, MD5, SHA1, DES, and other algorithms that were once considered as strong. Examples of currently recommended algorithms ([NSA](https://cryptome.org/2016/01/CNSA-Suite-and-Quantum-Computing-FAQ.pdf "Commercial National Security Algorithm Suite and Quantum Computing FAQ"), [NIST](http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-57pt1r4.pdf "NIST Special Publication 800-57")), :
```
Comment (Sjord) [inline Document/0x04g-Testing-Cryptography.md]: > "NIST Special Publication 800-57"

I think the title "Recommendation for Key Management" is more descriptive.

> )), :

I think the comma should be removed here.

File: Document/0x04g-Testing-Cryptography.md [hunk 3]
```diff
@@ -109,13 +101,6 @@ Periodically ensure that the cryptography has not become obsolete. Some older al
...
-##### Info
-- [1] Commercial National Security Algorithm Suite and Quantum Computing FAQ - https://cryptome.org/2016/01/CNSA-Suite-and-Quantum-Computing-FAQ.pdf
-- [2] NIST Special Publication 800-57 - http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-57pt1r4.pdf
-- [4] NIST recommendations (2016) - https://www.keylength.com/en/4/
-- [5] BSI recommendations (2017) - https://www.keylength.com/en/8/
-- [6] Sweet32 attack -- https://sweet32.info/
-
 ##### Tools
 - QARK - https://github.com/linkedin/qark
 - Mobile Security Framework - https://github.com/ajinabraham/Mobile-Security-Framework-MobSF
```
Comment (Sjord) [inline Document/0x04g-Testing-Cryptography.md]: Should these also be converted into markdown style links?

Comment (sushi2k) [inline Document/0x04g-Testing-Cryptography.md]: Good spot @Sjord. In order to stay consistent this should also be in the same style as the reference links.

**Comment by sushi2k** [comment]
Thanks for the input. Sorry, I didnt' see the PR earlier. I already did the changes to the crypto chapter. I merged it now with the existing content. Thanks for your work guys.

**Pull request merged**